src/Interfaces/ICompostAnimal.cs

```diff
@@ -0,0 +1,9 @@
+namespace Trestlebridge.Interfaces
+{
+    //An interface for any animal or plant processed for their compost
+    public interface ICompostAnimal
```

Comment:
Are these interfaces necessary to complete the tickets on this PR? If not, they shouldn't be on the PR. Planning ahead is great, but coding ahead can get you into tight spots-- you shouldn't be writing code unless it specifically pertains to the ticket(s) you're working on